tools/serve/serve.py

```diff
@@ -458,23 +485,32 @@ def add_mount_point(self, url_base, path):
             ("GET", "*.asis", handlers.AsIsHandler),
             ("GET", "/.well-known/origin-policy", handlers.PythonScriptHandler),
             ("*", "*.py", handlers.PythonScriptHandler),
-            ("GET", "*", handlers.FileHandler)
         ]
 
+        sub_url_base = url_base if url_base != "/" else ""
         for (method, suffix, handler_cls) in routes:
             self.mountpoint_routes[url_base].append(
                 (method,
-                 "%s%s" % (url_base if url_base != "/" else "", suffix),
+                 "%s%s" % (sub_url_base, suffix),
                  handler_cls(base_path=path, url_base=url_base)))
 
+        # Inject polyfill into all html tests.
```

Comment:
Yes, the polyfill won't be inserted into routes which come earlier. I'd like to rework the injection so that it could wrap other routes. I've added *.html and *.htm in the latest patch.

Reply:
Actually, I was able to do this rewrite relatively simply. Now wptserve wraps each of the existing routes with the injection handler if a polyfill is specified.